src/runtime/runtime.cs

```diff
@@ -2143,6 +2143,18 @@ internal static void Py_CLEAR(ref IntPtr ob)
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern int Py_AddPendingCall(IntPtr func, IntPtr arg);
 
+        [DllImport(_PythonDll, EntryPoint = "PyThreadState_SetAsyncExc", CallingConvention = CallingConvention.Cdecl)]
+        internal static extern int PyThreadState_SetAsyncExc37Windows(uint id, IntPtr exc);
```

Comment:
Why do you need overloads with `uint` and `int`? They are basically equivalent at bit level.

Reply:
I added them because of "Strictly, the thread-id attribute is unsigned only for Python >= 3.7 and we still support 3.6" comment above. Do you want me to remove them?

Reply:
@filmor on the binary side there's no difference. The caller, if necessary, can bitcast. Any thoughts?

Reply:
I have removed the int and long declarations for now.